randovania/games/samus_returns/logic_database/Area 6.txt

```diff
@@ -214,6 +276,7 @@ Extra - asset_id: collision_camera_037
   > Room Center
       Any of the following:
           Space Jump or After Area 6 - Omega Metroid
+          # Roll under the Omega
           Morph Ball and Movement (Intermediate)
```

Comment:
this feel slike it should be a template?

Reply:
Depends. It could be in the Omega template for dodging, but this is for crossing the room with the Omega active